### PR TITLE
#1460 Polish: Add pretty print icon to source tabs, remove ':formatted' from file name

### PR DESCRIPTION
**Pull request by darkwing**
prettyprint

### CODE REVIEW

**Comment by jasonLaster** [comment]
Thanks!

Two thoughts on UI:
* perhaps blue `fill: var(--something-here)` could be nice colors are in `variables.css`
* the left padding could perhaps be reduced a bit.

@clarkbw what do you think?

File: src/components/SourceTabs.js [hunk 1]
```diff
@@ -228,12 +229,14 @@ const SourceTabs = React.createClass({
 
     return dom.div(
       {
-        className: classnames("source-tab", { active }),
+        className: classnames("source-tab", { active }) +
```
Comment (jasonLaster) [inline src/components/SourceTabs.js]: you can use the classnames helper for this too :)

```js
classnames("source-tab", { 
  active,
  pretty: isPretty({ url: source.get("url") })
})
```

**Comment by clarkbw** [comment]
> @clarkbw what do you think?

Agreed on both points.  A light-ish blue would be my choice, but pick something available in the variables.

Great work! 👍

**Comment by darkwing** [comment]
Updated!

**Pull request merged**